src/routes/[type]/[id_primary]/[id_secondary]/[id_tertiary]/+page.svelte

```diff
@@ -1,14 +1,47 @@
 <script>
 	import { SourceEntities } from '$lib'
 
-	/** @type {Source} */
+	/** @type {PageData} */
 	export let data
+	const { source, parsed_encoding, nav_data } = data
+	
+	/**
+	 * @param {Reference} reference
+	 */
+	function create_reference_string({ id_primary, id_secondary, id_tertiary }) {
+		return `${id_primary} ${id_secondary}:${id_tertiary}`
+	}
 
-	const reference = `${data.id_primary} ${data.id_secondary}:${data.id_tertiary}`
+	/**
+	 * @param {Reference} reference
+	 */
+	function create_url({ type, id_primary, id_secondary, id_tertiary }) {
+		return `/${type}/${id_primary}/${id_secondary}/${id_tertiary}`
+	}
 </script>
 
-<div class="flex justify-center prose max-w-full">
-	<h2>{reference}</h2>
+<div class="flex justify-center max-w-full">
```

Comment:
maybe we could chat through the UI here?

I think I prefer something closer to the pagination in daisyui, which essentially means "joins"  (https://daisyui.com/components/pagination/). I find the following a little more appealing but I'm interested in discussing it further:

<img width="377" height="386" alt="image" src="https://github.com/user-attachments/assets/18645399-7e54-4a4d-b546-72479e896ff6" />

Additionally, a little more vertical separation between the nav and entities would be helpful, i.e., a little more bottom padding.